packages/contracts/hardhat.config.ts

```diff
@@ -21,6 +21,8 @@ import './tasks/validate-chugsplash-dictator'
 import './tasks/whitelist'
 import './tasks/withdraw-fees'
 import 'hardhat-gas-reporter'
+import '@primitivefi/hardhat-dodoc';
+import './tasks/validate-output'
```

Comment:
@smartcontracts can you please review this approach for using a hardhat plugin and the script itself? This should aim to raise warnings or errors (we need to be able to run it initially in warning mode until we fix all natspec comments) on all contracts that miss natspec comments on `external` and `public` functions (as a minimum, although there might be more we could output). 
I've included this in the scope because as good as docs are they might still end up containing missing documentation unless we enforce it in build.